keydb.conf

```diff
@@ -282,6 +282,9 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo yes
 
+# Retrieving "message of today" using CURL requests.
+enable-motd no
```

Comment:
Should be commented out, and rely on the default (which should be yes).
E.g.
# enable-motd yes

Reply:
Addressed in the latest commit.